pkg/printers/internalversion/printers.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// nodeLabelRole specifies the role of a node
 	nodeLabelRole = "kubernetes.io/role"
+
+	// schedulingPaused indicates a Pod is gated by one or more scheduling readiness gates.
```

Comment:
I would use blocked instead of pause.

Reply:
Agree.  Or SchedulingGated (it should be rare enough that calling it gated will help people find the field.

Reply:
Updated to `schedulingGated` which is more close to the new field name.